libs/notifications/src/workflows/usage-limits/usage-limits.workflow.ts

```diff
@@ -0,0 +1,47 @@
+import { workflow } from '@novu/framework';
+import { z } from 'zod';
+import { renderUsageLimitsEmail } from './email';
+
+export const usageLimitsWorkflow = workflow(
+  'usage-limits',
+  async ({ step, payload }) => {
+    await step.email(
+      'email',
+      async (controls) => {
+        return {
+          subject: controls.subject,
+          body: await renderUsageLimitsEmail(payload, controls),
+        };
+      },
+      {
+        controlSchema: z.object({
+          subject: z.string().default('You are approaching your usage limits'),
+          previewText: z.string().default('You have used {{payload.percentage}}% of your monthly events'),
+        }),
+      }
+    );
+
+    await step.inApp(
+      'in-app',
+      async (controls) => {
+        return {
+          title: controls.title,
+          body: controls.body,
+        };
+      },
+      {
+        controlSchema: z.object({
+          title: z.string().default('You are approaching your usage limits'),
+          body: z.string().default('You have used {{payload.percentage}}% of your monthly events'),
+        }),
+      }
```

Comment:
I would expect to see a type error here, it should be `subject`
```suggestion
      async (controls) => {
        return {
          subject: controls.subject,
          body: controls.body,
        };
      },
      {
        controlSchema: z.object({
          subject: z.string().default('You are approaching your usage limits'),
          body: z.string().default('You have used {{payload.percentage}}% of your monthly events'),
        }),
      }
```

Reply:
Hmm strange, none happened :(